config.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+require('dotenv').config({silent: true});
```

Comment:
I had set the env variables in the Makefile to to avoid the dependency on  this package.

However, I do think that this way makes it a bit easier for a developer that's not "make-friendly" to mess around.

I this repo, I think we should lean toward 'easy and friendly' rather than 'you must do it this way'. 

-Al